package.json

```diff
@@ -117,8 +118,16 @@
       "publish": false
     }
   },
+  "pnpm": {
+    "overrides": {
+      "@octokit/core": "^4.0.0"
+    },
+    "overrides-notes": {
```

Comment:
This is super nice. Is `override-notes` a baked-in pnpm thing or does it just ignore unknown keys?

Reply:
It ignores unknown keys - would be cool if this were standardized or something 🤔